src/screens/explore/AccommodationDetails/AccommodationDetails.tsx

```diff
@@ -73,6 +73,41 @@ const AccommodationDetails = ({ route, navigation }: Props) => {
     }
   };
 
+  const amenitiesBadges = useMemo(() => {
+    if (accommodation?.amenities) {
+      const { otherAmenities, id, accommodationId, ...rest } = accommodation?.amenities;
+      const amenities = Object.keys(rest);
+      return amenities.map((amenity) => {
+        if (rest[amenity as keyof typeof rest]) {
+          const { icon, text, iconSet } =
+            AMENITIES_CHIP_DATA[amenity as keyof typeof AMENITIES_CHIP_DATA];
+          return (
+            <ThemedView key={amenity} style={styles.badge}>
+              <Icon name={icon} size={20} iconSet={iconSet} />
+              <Text style={styles.badgeText}>{text}</Text>
+            </ThemedView>
+          );
+        }
+      });
+    }
+  }, [accommodation]);
+
+  const otherAmenitiesBadgesMemo = useMemo(() => {
+    if (accommodation?.amenities) {
+      const separator = ', ';
+      if (accommodation?.amenities?.otherAmenities) {
```

Comment:
same here, let's create a function `getExtraAmenityBadges`

Reply:
added functions to the utils